tutorials/mongoose/query-twice.md

```diff
@@ -0,0 +1,33 @@
+This issue typically happens when the dev uses `await` and passes a callback.
```

Comment:
Mongoose throws a "Query was already executed" error when a given query is executed twice.
The most common explanation for is you're mixing `await` and callbacks.